terraform/azure-mssql/bind-mssql.tf

```diff
@@ -34,26 +34,14 @@ resource "random_password" "password" {
 }
 
 resource "null_resource" "create-sql-login" {
-
   provisioner "local-exec" {
-    command = format("psqlcmd %s %d %s master \"CREATE LOGIN [%s] with PASSWORD='%s'\"",
-                     var.mssql_hostname,
-                     var.mssql_port,
-                     var.admin_username,
-                     random_string.username.result,
-                     random_password.password.result)
-    environment = {
-      MSSQL_PASSWORD = var.admin_password
-    }
-  }
-
-  provisioner "local-exec" {
-	when = destroy
+	  when = destroy
     command = format("psqlcmd %s %d %s master \"DROP LOGIN [%s]\"",
```

Comment:
Though I'm not generally a fan of comments (because the names should reflect what we are trying to do), I was thinking that a comment here might be helpful as it does look a little strange to try and delete a resource that we didn't create.